Source/DafnyCore/Dafny.atg

```diff
@@ -3762,8 +3762,12 @@ ConstAtomExpression<out Expression e>
     "|"
   | "assigned"                                 (. x = t; .)
     "(" NameSegment<out e> ")"                 (. e = new UnaryOpExpr(x, UnaryOpExpr.Opcode.Assigned, e); .)
-  | IF(AcceptReferrersAndBacktick())           (. e = new ImplicitThisExpr(la); .)
-    FieldLocationSuffix<ref e>
+  | "`"                                        (. if (!AcceptReferrers()) {
```

Comment:
Neat. I would not have guess that this would work since "backtick name" is always a valid frame expression.
But it works great !